build.gradle

```diff
@@ -304,7 +304,7 @@ sonarqube {
     property "sonar.jacoco.reportPath", "${listFiles('**/test.exec')}"
     property "sonar.jacoco.itReportPath", "${listFiles('**/apiTest.exec')}"
     property "sonar.exclusions", "**/ccd/domain/**," +
-      "**/ccd/jackson/mixin/**," +
+      "**/ccd/mapper/**," +
```

Comment:
Indeed we lack of test coverage hence Sonar would complain rightfully if we tried to merge. 

However given the size of this PR which has already grown out of proportion I would suggest to retrofit all missing tests in the next PR and leave this exclusion rule temporarily for one day or two. I took a note to remind ourselves to reinstate the rule in next PR.

Reply:
I do not agree to retrofitting unit tests . That basically throws TTD out of the window.

This being a big PR is our own doing .It didn't have to be this way. There were stories broken down for a reason but the dev still chose to club all the stories and make this a big PR, which is not correct. 

Most importantly, just because a PR is big , it doesn't mean we should rush to merge it. Infact we should be doing a more thorough review.

Reply:
Removed the exclusion.

Reply:
👍